src/main/resources/liquibase/harness/change/expectedSql/mariadb/10.2/alterSequenceCycleProperty.sql

```diff
@@ -0,0 +1,3 @@
+INVALID TEST
+
```

Comment:
It's tricky. If we don't support MariaDB in general, than tests for other platforms would fail as well. But for 10.3 and higher it works.
Documentation is also ambiguous.
https://docs.liquibase.com/change-types/community/alter-sequence.html
"Database support" section says we don't support MariaDB, but "Available attributes" section say half attributes are supported.